app/models/service_provider.rb

```diff
@@ -30,6 +30,7 @@ class ServiceProvider < ApplicationRecord
 
   ALLOWED_IAL1_ATTRIBUTES = %w[
     email
+    all_emails
```

Comment:
**question:** do we not need to update any of our tests?

Reply:
Too fast, I just posted this! :)